community/neovim/depends

```diff
@@ -0,0 +1 @@
+cmake
```

Comment:
Should be `cmake make` as I assume `cmake` isn't needed at runtime (*only at compile time*).

Reply:
Fixed in f19fb7f